airbyte-cdk/python/airbyte_cdk/sources/declarative/declarative_component_schema.yaml

```diff
@@ -903,6 +903,31 @@ definitions:
             default: ["credentials", "token_expiry_date"]
             examples:
               - ["credentials", "token_expiry_date"]
+          refresh_token_error_status_codes:
+            title: Refresh Token Error Status Codes
+            description: Status Codes to Identify refresh token error in response.
+            type: array
+            items:
+              type: integer
+            default: []
+            examples:
+              - [400, 500]
+          refresh_token_error_key:
+            title: Refresh Token Error Key
+            description: Key to Identify refresh token error in response.
+            type: string
+            default: ""
+            examples:
+              - "error"
+          refresh_token_error_values:
+            title: Refresh Token Error Values
+            description: 'List of values to check for exception during token refresh process. Used to check if the error found in the response matches the key from the Refresh Token Error Key field (e.g. response={"error": "invalid_grant"}). Raised AirbyteTracedException configuration error.'
```

Comment:
@artem1205 I don't understand how this is meant to be used. From looking at the code, it looks like only responses with one of the error status and containing an error value will be flagged as a config error. Is this correct?

Can you clarify so users don't need to look at the code to understand how this is meant to be used?

Reply:
> From looking at the code, it looks like only responses with one of the error status and containing an error value will be flagged as a config error. Is this correct?

You're right.

> Can you clarify so users don't need to look at the code to understand how this is meant to be used?

Description updated